erpnext/payroll/doctype/payroll_entry/payroll_entry.js

```diff
@@ -17,6 +17,16 @@ frappe.ui.form.on('Payroll Entry', {
 				}
 			};
 		});
+
+		frm.set_query("payroll_payable_account", function() {
+			return {
+				filters: {
+					"company": frm.doc.company,
+					"root_type": "Liability",
+					"is_group": 0,
```

Comment:
"account_currency": frm.doc.currency

Reply:
We were allowing it to be in either employee's currency or the company's currency.